papaya-core/src/main/java/pl/edu/agh/papaya/service/sprint/SprintService.java

```diff
@@ -96,6 +89,14 @@ Sprint createSprint(Sprint sprint) {
 
     public Sprint closeSprint(Sprint sprint, Duration timeBurned, Duration estimatedTimePlanned,
             Duration finalTimePlanned, LocalDateTime dateClosed) {
+        if (sprint.getDateClosed() != null) {
```

Comment:
Ewaluację argumentów od użytkownika wyjęłam do `SprintsRestService`, ale to nie wiem, gdzie chcemy